tests/test_mfn.py

```diff
@@ -0,0 +1,23 @@
+"""Tests for the MFn bindings."""
+
+import cmdc
+import maya.api.OpenMaya
+
+def test_fn_types():
+    """Check that the Fn.Type enum is identical to the MFn.Type enum."""
+
+    for k, v in maya.api.OpenMaya.MFn.__dict__.items():
+        if not k.startswith('k'):
+            continue 
+
+        assert hasattr(cmdc.Fn.Type, k), "Missing enum '%s'." % k
```

Comment:
Oh this is a *great* idea. We'll be able to get really accurate consistency with this. If wonder if we could do something similar for fetching docstrings, and maybe even argument types? Maybe there are ways of automating much of this binding after all? :O